photo-helper/src/main/java/io/intrepid/photohelper/ContextDelegate.java

```diff
@@ -0,0 +1,17 @@
+package io.intrepid.photohelper;
+
+import android.content.ContextWrapper;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+
+/**
```

Comment:
This comment results in a lint warning ("Default file template") on my box - can you double-check this and either reformat it or suppress the warning?